tests/cypress/e2e/helpers.ts

```diff
@@ -25,6 +17,17 @@ export function createCmkAutomationUser(cmkUser: string, cmkPassword: string) {
   });
 }
 
+export function deleteCmkAutomationUser(cmkUser: string, cmkPassowrd: string, failOnStatusCode: boolean = false) {
```

Comment:
default for `failOnStatusCode` should be true? you explicitly pass `false`.

Reply:
Right. I fixed it in the latest commit. Thanks for the catch!